### PR TITLE
Docs: added documentation for `yt-dlp` feature

### PR DESCRIPTION
**Pull request by zunda-arrow**
I noticed the code for this feature is in the current branch but the documentation for it is missing.

### CODE REVIEW

File: src/input/ytdl_src.rs [hunk 1]
```diff
@@ -31,8 +31,9 @@ const YOUTUBE_DL_COMMAND: &str = if cfg!(feature = "youtube-dlc") {
 /// If you need looping or track seeking, then consider using
 /// [`Restartable::ytdl`].
 ///
-/// Uses `youtube-dlc` if the `"youtube-dlc"` feature is enabled.
-///
+/// `youtube-dlc` and `yt-dlp` are also available by enabling the `youtube-dlc`
+/// and `yt-dlp` features respectively.
+///
```
Comment (FelixMcFelix) [inline src/input/ytdl_src.rs]: You've left a trailing space on this line: either remove it, or just run `cargo +nightly fmt`.

**Review by FelixMcFelix — CHANGES_REQUESTED** [review]
One minor lint in the doc comment, then it's good to go!

File: src/input/ytdl_src.rs (hunk 1 above)
Comment (zunda-arrow) [reply src/input/ytdl_src.rs]: I've fixed it xD

**Review by FelixMcFelix — APPROVED** [review]
Perfect, thanks!

**Pull request merged**